sentry/templates/deployment-sentry-post-process-forwarder.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sentry.prefix" . }}post-process-forward
+  labels:
+    app: sentry
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  selector:
+    matchLabels:
+        app: sentry
+        release: "{{ .Release.Name }}"
+        role: sentry-post-process-forward
+  replicas: {{ .Values.sentry.post_process_forward.replicas }}
+  template:
+    metadata:
+      annotations:
+        checksum/configYml: {{ .Values.config.yaml | sha256sum }}
+        checksum/sentryConfPy: {{ .Values.config.py | sha256sum }}
+        checksum/config.yaml: {{ include (print $.Template.BasePath "/configmap-sentry.yaml") . | sha256sum }}
+      labels:
+        app: sentry
+        release: "{{ .Release.Name }}"
+        role: sentry-post-process-forward
+        {{- if .Values.sentry.post_process_forward.podLabels }}
+{{ toYaml .Values.sentry.post_process_forward.podLabels | indent 8 }}
+        {{- end }}
+    spec:
+      affinity:
+      {{- if .Values.sentry.post_process_forward.affinity }}
+{{ toYaml .Values.sentry.post_process_forward.affinity | indent 8 }}
+      {{- end }}
+      {{- if .Values.sentry.post_process_forward.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.sentry.post_process_forward.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.sentry.post_process_forward.tolerations }}
+      tolerations:
+{{ toYaml .Values.sentry.post_process_forward.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.images.sentry.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.images.sentry.imagePullSecrets | indent 8 }}
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}-post-process-forward
+        image: "{{ .Values.images.sentry.repository }}:{{ .Values.images.sentry.tag }}"
+        imagePullPolicy: {{ .Values.images.sentry.pullPolicy }}
+        command: ["sentry", "run", "post-process-forwarder", "--commit-batch-size 1"]
+        env:
+          - name: SNUBA
+            value: http://{{ template "sentry.prefix" . }}snuba:{{ template "snuba.port" }}
+        {{ if eq .Values.filestore.backend "gcs" }}
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /var/run/secrets/google/{{ .Values.filestore.gcs.credentialsFile }}
+        {{ end }}
+{{- if .Values.sentry.post_process_forward.env }}
+{{ toYaml .Values.sentry.post_process_forward.env | indent 8 }}
+{{- end }}
+        volumeMounts:
+        - mountPath: /etc/sentry
+          name: config
+          readOnly: true
+        - mountPath: {{ .Values.filestore.filesystem.path }}
```

Comment:
Should we really mount the volume in the path ? If it doesn't support ReadWriteMany, the container won't be able to start

Reply:
https://github.com/getsentry/onpremise/blob/master/docker-compose.yml

Here I can see they are mounting sentry-data in each container, even in post process forwarder, right?